setup.py

```diff
@@ -13,6 +13,13 @@
     'Operating System :: POSIX :: Linux',
     'Programming Language :: C',
     'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3 :: Only',
```

Comment:
memo: current cgroup-utils still supports Python 2 but the next release will drop it.  So the declaration is valid.